lib/cloudregister/smt.py

```diff
@@ -33,6 +33,10 @@ def __init__(self, smtXMLNode, https_only=False):
             self._ipv6 = smtXMLNode.attrib['SMTserverIPv6']
         except KeyError:
             pass
+        try:
+            self._region = smtXMLNode.attrib['region']
+        except KeyError: 
```

Comment:
there's a trailing space after the colon, not sure why it's not highlighted by github web